config/_default/config.toml

```diff
@@ -1,4 +1,4 @@
-baseurl = "https://edu.chainguard.dev/"
+baseUrl = "https://edu.chainguard.dev/"
 canonifyURLs = false
```

Comment:
I am working on analytics related to canonifying URLs, but we may want to revisit documentation here

https://gohugo.io/content-management/urls/#canonicalization

Reply:
yeah I thought this should be set to true only for production, however I saw this on the docs: `In the May 2014 release of Hugo v0.11, the default value of canonifyURLs was switched from true to false, which we think is the better default and should continue to be the case going forward`

so I don't know anymore. In any case if we decide to change it I think it should go on the config/production folder